src/components/Layout/Layout.module.css

```diff
@@ -0,0 +1,87 @@
+.newLayout {
+  --black: lab(0.07 0.35 0.12);
+  --dark-grey: lab(43.19 0 0);
+  --grey: lab(66.99 0 0);
+  --light-grey: lab(94.10 0 0);
+  --off-white: lab(100 0 0);
+
+  --blue: lab(49.827 31.172 -78.866);
+
+  --background-color: var(--light-grey);
+  --text-color: var(--black);
+  --link-color: var(--dark-grey);
+
+  --wrapper-width: 1192px;
+
+  &.isAfterDark {
+    --background-color: var(--black);
+    --text-color: var(--off-white);
+    --link-color: var(--grey);
+  }
+}
+
+.newLayout {
+  @mixin body-4;
+  background-color: var(--background-color);
+  color: var(--text-color);
+  width: 100%;
+  padding: 0 20px;
+
+  @media (--tablet) {
+    @mixin body-2;
+    padding: 0 40px;
+  }
+
+  & .newLayout {
+    padding: 0;
+  }
```

Comment:
`.newLayout & .netLayout` ? why are these nested?

Reply:
It's a workaround because we don't apply the newLayout to all pages, but the header and footer **do** require the newLayout. However newLayout also handles standardised padding, therefore we need to clear this to avoid double padding on header/footer on studio page.

This can be stripped out once newLayout is standard everywhere (in fact newLayout will just become :root)

Reply:
Will do